### PR TITLE
Making INDEX_PATH, RSS_PATH and AUTHOR_PATH translatable

### LINKED ISSUE #1914: Translatable URL pathnames
Would be nice if we could have translatable *_PATH names in config file.
Think of  /en/categories/cars and /es/categorias/automovil on the same site.

https://groups.google.com/forum/#!topic/nikola-discuss/LXuI3c6eq2Q
- [x] TAG_PATH
- [x] CATEGORY_PATH
- [ ] AUTHOR_PATH
- [ ] INDEX_PATH
- [ ] RSS_PATH

### PR DESCRIPTION
**Pull request by felixfontein**
The last three checkmarks for #1914.

(fixes #1914)

### CODE REVIEW

File: nikola/plugins/task/authors.py [hunk 1]
```diff
@@ -73,15 +73,15 @@ def get_classification_friendly_name(self, author, lang, only_last_component=Fal
 
     def get_overview_path(self, lang, dest_type='page'):
         """A path handler for the list of all classifications."""
-        return [self.site.config['AUTHOR_PATH']], 'always'
+        return [self.site.config['AUTHOR_PATH'][lang]], 'always'
```
Comment (Kwpolska) [inline nikola/plugins/task/authors.py]: Always call TranslatableSettings with `(lang)`, never index them.

File: nikola/conf.py.in [hunk 2]
```diff
@@ -389,6 +389,7 @@ HIDDEN_CATEGORIES = []
 # output / TRANSLATION[lang] / AUTHOR_PATH / index.html (list of tags)
```
Comment (Kwpolska) [inline nikola/conf.py.in]: I know this was there before, but while you’re at it, please change this to:

```
# Path to author pages. Final locations are:
# output / TRANSLATION[lang] / AUTHOR_PATH / index.html (list of authors)
# output / TRANSLATION[lang] / AUTHOR_PATH / author.html (list of posts by an author)
# output / TRANSLATION[lang] / AUTHOR_PATH / author.xml (RSS feed for an author)
# (translatable)
```

File: nikola/plugins/task/authors.py (hunk 1 above)
Comment (felixfontein) [inline nikola/plugins/task/authors.py]: Fixed.

File: nikola/conf.py.in (hunk 2 above)
Comment (felixfontein) [inline nikola/conf.py.in]: Fixed.

**Pull request merged**